.gitignore

```diff
@@ -8,19 +8,21 @@
 /web/libraries/
 /web/profiles/config_installer/
 
-# Ignore Drupal's file directory.
-/web/sites/*/files/
```

Comment:
This gitignore was moved to `/web/sites/.gitignore` to solve an issue with Pantheon.